test/test.py

```diff
@@ -516,14 +516,14 @@ def _assert_bundle(self, uuid, version, manifest, metadata_files,
         return bundle
 
     def test_canned_staging_area(self):
-        commit = '2c7fbd0e0b7804abeb21ddb0ae89724950218257'
-        url = f'https://github.com/HumanCellAtlas/schema-test-data/tree/{commit}/tests'
-        sa_factory = GitHubStagingAreaFactory.from_url(url)
-        sa = sa_factory.load_staging_area()
-        self.assertGreater(len(sa.links), 0)
-        for link_id in sa.links:
+        ref = '2c7fbd0e0b7804abeb21ddb0ae89724950218257'  # release-data-26%2F03%2F2021
```

Comment:
We can't merge this PR until they merge theirs and we update this to point at the merge commit on their default branch.

Reply:
https://github.com/HumanCellAtlas/schema-test-data/pull/8 has been merged https://github.com/HumanCellAtlas/schema-test-data/commit/de355cad77ea7988040b6f1f5f2eafae58f686a8 .
I will set `ref` to `de355cad77ea7988040b6f1f5f2eafae58f686a8`